### PR TITLE
GSoC 2018 updates

### PR DESCRIPTION
**Pull request by omazapa**
Dear HSF,
this is a PR for GSoC 2018 with more mentors, organizations and proposals.

Best Regards,
Omar.

### CODE REVIEW

File: _activities/gsoc.md [hunk 1]
```diff
@@ -15,42 +15,58 @@ and analyze petabytes data from high-energy physics experiments, such as the Lar
 hosted at the CERN laboratory in Geneva, Switzerland. 
 Some of the questions that we collectively ask are: 
 
-- what are the fundamental blocks that make up our Universe? 
-- what is the nature of dark matter and dark energy?
-- what is the nature of the asymmetry between matter and antimatter? 
-- what was early Universe like? 
+- What are the fundamental blocks that make up our Universe?
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md]: I disagree with this change - the sentence is a *list* of questions, so the start of each bullet should not be capitalised. (i.e., it's "I have various questions: who? what? where? and why?").

File: _activities/gsoc.md [hunk 2]
```diff
@@ -15,42 +15,58 @@ and analyze petabytes data from high-energy physics experiments, such as the Lar
...
 
-Information from last year's GSoC can be found [here](/gsoc/2017/index.html). For 2018 the 
-HSF is again applying for participation in the program. If you are 
-interested in the GSoC program contact us using the HSF GSoC mailing list: [hep-software-foundation-google-summer-of-code@googlegroups.com](mailto:hep-software-foundation-google-summer-of-code@googlegroups.com).
+If you are interested in the GSoC program contact us using the HSF GSoC mailing list: [hep-software-foundation-google-summer-of-code@googlegroups.com](mailto:hep-software-foundation-google-summer-of-code@googlegroups.com)
 
 Instructions for participating projects and mentors can be found [here](/gsoc/guideline.html).
 
-## Projects in 2018
+## 2018 Projects
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md]: Now you introduced an asymmetry in the two titles for projects and institutes - we should do one or the other, but be consistent!

File: _activities/gsoc.md [hunk 3]
```diff
@@ -15,24 +15,23 @@ and analyze petabytes data from high-energy physics experiments, such as the Lar
...
 Some of the questions that we collectively ask are: 
 
-- what are the fundamental blocks that make up our Universe? 
-- what is the nature of dark matter and dark energy?
-- what is the nature of the asymmetry between matter and antimatter? 
-- what was early Universe like? 
+- What are the fundamental blocks that make up our Universe?
+- What is the nature of dark matter and dark energy?
+- What is the nature of the asymmetry between matter and antimatter?
+- What was the early Universe like?
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md]: That was a mistake - thanks!

File: _activities/gsoc.md [hunk 4]
```diff
@@ -54,14 +53,15 @@ Instructions for participating projects and mentors can be found [here](/gsoc/gu
...
 | ![Karlsruhe Institute of Technology](/images/kit_logo.png){:width="100px"} | The Karlsruhe Institute of Technology is a public research university and one of the largest research and education institutions in Germany. [List of proposals](/gsoc/2018/kit.html)|
 | ![University of Florida](/images/ufl_logo.jpg){:width="100px"} | University of Florida is a public institution that was founded in 1853. [List of proposals](/gsoc/2018/uflorida.html)|
 | ![Florida State University](/images/fsu_logo.jpg){:width="100px"} | Florida State University is a public institution that was founded in 1851. [List of proposals](/gsoc/2018/fsu.html)|
-| ![Lulea University of Technology](/images/lut_logo.jpg){:width="100px"} | (Swedish: Luleå tekniska universitet) of Sweden is Scandinavia's northernmost university of technology. [List of proposals](/gsoc/2018/lut.html)|
+| ![Lulea University of Technology](/images/lut_logo.jpg){:width="100px"} | Lulea University of Technology (Swedish: Luleå tekniska universitet) of Sweden is Scandinavia's northernmost university of technology. [List of proposals](/gsoc/2018/lut.html)|
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md]: Lulea don't seem to have any proposals? Is there something missing?

File: _activities/gsoc.md [hunk 5]
```diff
@@ -15,42 +15,58 @@ and analyze petabytes data from high-energy physics experiments, such as the Lar
...
+| ![Karlsruhe Institute of Technology](/images/kit_logo.png){:width="100px"} | The Karlsruhe Institute of Technology is a public research university and one of the largest research and education institutions in Germany. [List of proposals](/gsoc/2018/kit.html)|
 | ![LAL](/images/logo_LAL.jpg){:width="100px"} | [LAL](http://www.lal.in2p3.fr) is a French research laboratory belonging to CNRS/IN2P3 and located at Université Paris Sud. The main topics of the research done at LAL are high energy physics, cosmology and accelerators. [List of proposals](/gsoc/2018/lal.html)|
 | ![LPC](/images/lpc-logo.png){:width="100px"} | Founded in 1958, Laboratoire de Physique de Clermont is a government-funded mixed research unit (CNRS/IN2P3 and University Clermont Auvergne), in Auvergne, France. [List of proposals](/gsoc/2018/lpc-clermont.html)|
+| ![Lulea University of Technology](/images/lut_logo.jpg){:width="100px"} | Lulea University of Technology (Swedish: Luleå tekniska universitet) of Sweden is Scandinavia's northernmost university of technology. [List of proposals](/gsoc/2018/lut.html)|
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md]: There are no project proposals from Lulea - something missing?

File: _activities/gsoc.md [hunk 6]
```diff
@@ -15,42 +15,58 @@ and analyze petabytes data from high-energy physics experiments, such as the Lar
...
+| ![TMVA](/images/tmva_logo.gif){:width="100px"} | Toolkit for Multivariate Analysis (TMVA) is a multi-purpose machine learning toolkit integrated into the ROOT scientific software framework, used in many particle physics data analysis and applications. [List of proposals](/gsoc/2018/project_TMVA.html)|
 
 ## Participating Organizations in 2018
 
 {:.table .table-hover  .table-striped}
 | ![CERN](/images/CERN-logo.jpg){:width="100px"} | At CERN, the European Organization for Nuclear Research, physicists and engineers are probing the fundamental structure of the universe. They use the world's largest and most complex scientific instruments to study the basic constituents of matter – the fundamental particles. [List of proposals]({{site.baseurl}}/gsoc/2018/cern.html)|
+| ![École polytechnique fédérale de Lausanne](/images/Logo_EPFL.png){:width="100px"} | The École polytechnique fédérale de Lausanne (EPFL) is a research institute and university in Lausanne, Switzerland, that specializes in natural sciences and engineering. [List of proposals](/gsoc/2018/epfl.html)|
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md]: There are no project proposals from EPFL - something missing?

File: _activities/gsoc.md [hunk 7]
```diff
@@ -15,42 +15,58 @@ and analyze petabytes data from high-energy physics experiments, such as the Lar
...
 | ![LAL](/images/logo_LAL.jpg){:width="100px"} | [LAL](http://www.lal.in2p3.fr) is a French research laboratory belonging to CNRS/IN2P3 and located at Université Paris Sud. The main topics of the research done at LAL are high energy physics, cosmology and accelerators. [List of proposals](/gsoc/2018/lal.html)|
 | ![LPC](/images/lpc-logo.png){:width="100px"} | Founded in 1958, Laboratoire de Physique de Clermont is a government-funded mixed research unit (CNRS/IN2P3 and University Clermont Auvergne), in Auvergne, France. [List of proposals](/gsoc/2018/lpc-clermont.html)|
+| ![Lulea University of Technology](/images/lut_logo.jpg){:width="100px"} | Lulea University of Technology (Swedish: Luleå tekniska universitet) of Sweden is Scandinavia's northernmost university of technology. [List of proposals](/gsoc/2018/lut.html)|
+| ![Moscow Institute of Physics and Technology](/images/MIPT_logo.jpg){:width="100px"} | Moscow Institute of Physics and Technology  is a Russian university, originally established in Soviet Union. It prepares specialists in theoretical and applied physics, applied mathematics, and related disciplines. [List of proposals](/gsoc/2018/mipt.html)|
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md]: No project proposals...

**Comment by omazapa** [comment]
Thank you very much Graeme for your comments.
Noted about the capitalization (fixed). 
The asymmetry is also fixed. 
Regarding institutes, if a mentor is listed in any of the projects then their institute is added to the main page. For example, Vladimir Ilievski (EPFL), Kim Albertsson (LUT) are all mentors for TMVA projects (Additional TMVA projects), and Dimitri Sorokin (MIPT) is a mentor for Geant.

Best Regards
Omar.

**Comment by omazapa** [comment]
Hi Guys, 
the problem appear to be a problem in travis,
maybe running travis again can be fixed. (connection timed out)


 Failed to fetch http://ppa.launchpad.net/openjdk-r/ppa/ubuntu/dists/trusty/main/binary-amd64/Packages  Could not connect to ppa.launchpad.net:80 (91.189.95.83), connection timed out
 *  External link https://indico.hephy.oeaw.ac.at/event/86 failed: got a time out (response code 0)
...
Cheers,

**Comment by hegner** [comment]
Be careful here with the addition of institutes - for MIPT for example we are waiting for a go-ahead on whether we are allowed to put their name and logo. That a member of a university contributes doesn't mean the university itself endorses the HSF GSoc.

**Comment by jouvin** [comment]
@omazapa  Also I opened #233 to streamline addition of projects and organizations on the main page. It would be good to have it merged first and to have this PR rebased on it (location of project/organization files have been changed and a few additional attributes are needed in the front-matter section).

**Comment by hegner** [comment]
@omazapa - I will take care and create a new PR

**Comment by hegner** [comment]
Now handled via #236

**Pull request closed without merging**